src/Generator/TypeGenerator.php

```diff
@@ -121,4 +152,9 @@ private static function isInternalPhpType($type)
     {
         return in_array(strtolower($type), self::$internalPhpTypes, true);
     }
+
+    private function isAlias()
```

Comment:
No need for a private function for this: the state is private anyway, so you can access it directly.